pkgs/by-name/gi/gitlab/rubyEnv/Gemfile

```diff
@@ -333,9 +333,6 @@ gem 'connection_pool', '~> 2.4', feature_category: :shared
 # Redis session store
 gem 'redis-actionpack', '~> 5.5.0', feature_category: :redis
 
-# Discord integration
-gem 'discordrb-webhooks', '~> 3.5', require: false, feature_category: :integrations
-
```

Comment:
The discord integration "could cause DoS" according to the release notes. Is the upstream mitigation for the issue to remove the integration?